extension/js/fill_relay_address.js

```diff
@@ -44,12 +44,12 @@ async function showModal(modalType, newAlias=null) {
 
   if (modalType === "max-num-aliases") { // User has maxed out the number of allowed free aliases.
     const modalMessage = document.createElement("span");
-    modalMessage.textContent = "You have reached the maximum number of aliases allowed during the beta phase of Private Relay.";
+    modalMessage.textContent = "You've used all of your beta aliases.";
     modalMessage.classList = ["relay-modal-message relay-modal-headline"];
     modalContent.appendChild(modalMessage);
 
     const manageAliasesLink = document.createElement("a");
-    manageAliasesLink.textContent = "Manage Relay Addresses";
+    manageAliasesLink.textContent = "Manage All Addresses";
     manageAliasesLink.classList = ["new-tab"];
     manageAliasesLink["href"] = `${relaySiteOrigin}?utm_source=fx-relay-addon&utm_medium=context-menu-modal&utm_campaign=beta&utm_content=manage-relay-addresses`;
```

Comment:
unrelated, but why is this using object notation for `["href"]` here, but dot notation elsewhere for `.textContent` and `.classList`?

Reply:
If I had a good reason, I don't remember it. Updated.